pipeline/unified/build-unsigned-release-packages.yaml

```diff
@@ -23,7 +23,7 @@ steps:
 
     - template: ../electron/electron-e2e-publish-results.yaml
 
-    - script: yarn build:electron:all --unified-version=$(Build.BuildNumber) --app-insights-instrumentation-key=$(AppInsightsInstrumentationKey)
+    - script: yarn build:electron:all --unified-version=$(Build.BuildNumber) --unified-canary-instrumentation-key$(UnifiedCanaryInstrumentationKey) --unified-insider-instrumentation-key=$(UnifiedInsiderInstrumentationKey) --unified-prod-instrumentation-key=$(UnifiedProdInstrumentationKey)
```

Comment:
missing = sign in `--unified-canary-instrumentation-key$(UnifiedCanaryInstrumentationKey)`?

Reply:
added! Thanks